executor/executor.cc

```diff
@@ -548,6 +561,12 @@ void receive_execute()
 	}
 	if (req.prog_size == 0)
 		fail("need_prog: no program");
+#if GOOS_windows
+	DWORD dwRead = 0;
+	if (!ReadFile((HANDLE)_get_osfhandle(249), input_data, req.prog_size, &dwRead, NULL)) {
```

Comment:
s/249/kInPipeFd/

Reply:
Okay Will use the variables rather than the value.